frontend/src/App.jsx

```diff
@@ -7,13 +7,13 @@ import { getAuth, signOut, signInWithPopup, GoogleAuthProvider } from "firebase/
 const provider = new GoogleAuthProvider();
 
 const firebaseConfig = {
-  apiKey: "AIzaSyACtvwa5uXyqLiUrGL_7YsrzdJMUpg6YYQ",
```

Comment:
@devintor not super important to this project but for the future - consider that the secret is now leaked to GitHub. Even though you added a new commit to remove it, it's still here in the version history.

before launching this app to the public, you should create a new FireBase account and update the app to use the secret from that account.